app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @posts= Post.find.(params(:id))
```

Comment:
This will error out. It should be `Post.find(params[:id])` (with no period after `find`).